AutomatedTesting/Gem/PythonTests/Terrain/EditorScripts/TerrainPhysicsCollider_MaterialMapping_Works.py

```diff
@@ -55,11 +55,15 @@ def TerrainPhysicsCollider_MaterialMapping_Works():
     rubber_distance = rubber_finish.GetDistance(rubber_start)
 
     # Report results on distance traveled and validate that the Cube on the glass surface travels further
+    expected_glass_distance = 7.0
```

Comment:
Might want to bump this down to 5.0 to be even more safe.

Reply:
Will bump down for now, and update to check for resting state in a later check-in. That will likely require some updates to the premade level, as the cube will slide off the edge of the terrain after enough time as is.

Reply:
> It would be worth checking with the physics folks if there's a way to tell when these cubes have reached a resting state, instead of waiting N seconds, so that you're less dependent on processor speed for the final result.

#8200 addresses this.